config/initializers/delayed_job_config.rb

```diff
@@ -32,3 +32,5 @@
 
 # By default bypass worker queue and execute asynchronous tasks at once
 Delayed::Worker.delay_jobs = !Rails.env.test?
+
+Delayed::Worker.logger = Logger.new(File.join(Rails.root, 'log', 'delayed_job.log'))
```

Comment:
Isn't this the default logging location of delayed job? I'm just asking since this is probably something that deployment would have to address (/cc @jonasheinrich)